.ci/messages-acceptable-words_xx.properties

```diff
@@ -0,0 +1,3604 @@
+aaaa
+AAAAA
+aaaaaa
+AAAAAAA
+aab
+aac
+aad
+aae
+aaf
+aar
+aarch
+aasdf
+Abbrechen
+abbreviationaswordinname
+ABCD
+ABCDEFl
+abcun
+abe
+abego
+abf
+abstractcheck
+abstractclassname
+abstractjavadoc
+abstracttypeaware
+Abulfaz
+ABx
+aca
+acb
+Acceptabletokens
+acd
+acf
+Acheckstyle
+Aclass
+acm
+Aconfig
+ada
+adb
+Addables
+addons
+adf
+adr
+aea
+aeb
+aec
+aed
+aee
+aef
+aeiou
+afa
+afc
+AFDF
+afe
+AFilter
+AFp
+aggregators
+Agoogle
+Ahmadov
+akhil
+Akshita
+alberto
+aleksey (Aleksey, aleksey)
+alex
+Alexey
+allchecks
```

Comment:
Why does `allchecks` have to be here? This isn't a 'word' by itself.
I assume original place had it in camel case like `AllChecks`?

Reply:
```
src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java:        return "com/puppycrawl/tools/checkstyle/internal/allchecks";
src/test/resources/com/puppycrawl/tools/checkstyle/internal/allchecks/package-info.java:package com.puppycrawl.tools.checkstyle.internal.allchecks;
src/test/resources/com/puppycrawl/tools/checkstyle/internal/allchecks/InputAllChecksDefaultConfig.java:package com.puppycrawl.tools.checkstyle.internal.allchecks;
```

Reply:
It might be best not to call these things "words", since they aren't precisely words so much as ... well, something. I'm open to suggestions.

Reply:
ok, it is because of the package name, which is not in camel case.